charts/node-red/values.yaml

```diff
@@ -1,132 +1,28 @@
 # Default values for node-red.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
-# upgrade strategy type (e.g. Recreate or RollingUpdate)
-strategyType: Recreate
 
 image:
   repository: nodered/node-red
-  tag: 1.0.6-12-minimal
   pullPolicy: IfNotPresent
+  tag: 1.2.5
```

Comment:
I don't know what the reasoning used to be to go for the `minimal` variant. But it also makes sense to me to go for the 'default' image by default and if people want to use another variant they can override the value

Reply:
agreed!